packages/node/test/e2e/requester-fulfill.feature.ts

```diff
@@ -68,6 +57,7 @@ it('should call fail function on AirnodeRrp contract and emit FailedRequest if r
 
   const postInvokeLogs = await fetchAllLogs(provider, deployment.contracts.AirnodeRrp);
   expect(postInvokeLogs.map(({ name }) => name)).toEqual([
+    'SetSponsorshipStatus', // RrpRequester constructor
```

Comment:
Same here, this should be changed with `expect.arrayContaining(postInvokeExpectedLogs)`